### PR TITLE
feat: changelog for 2.0.0

### PR DESCRIPTION
**Pull request by inckie**
Changelog for 2.0.0

### Checklist
- [x] I've provided a readable title for the PR
- [x] I've provided a detailed description
- [x] The PR is scoped to a single task/feature
- [ ] I've included relevant tests (if tests are not included please provide the reason why they aren't)

- [x] My PR is not introducing a UI change
- [ ] My PR is introducing UI changes and I provided all screenshots in the PR description

- [x] My PR is a part of a new feature or a feature improvement
    - [ ] I've provided a link in the description to the relevant card in the Zapp cycle feature rollout Trello board.
- [ ] My PR is a bug fix
    - [ ] I provided a link in the description to the relevant Jira ticket  or provided the following in the PR description:
        - steps to reproduce a bug
        - expected result

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+#### Minimum Android OS supported: 4.4.x
+#### React Native version: 0.62.2
+#### Gradle version: 5.6.4
+#### Android Studio Gradle Plugin version: 3.5.3
+#### QuickBrick version: v4.1.4
+#### Applicaster Android SDK Core version: 8.0.1
+#### Android target API: 29
+
+### Improvements:
+- RtL support improved [#66](https://github.com/applicaster/zapp-platform-android/pull/66)
```
Comment (ranmeirman) [inline CHANGELOG.md:15]: Can you elaborate a bit?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+- Firebase push notification DeepLink support added [#69](https://github.com/applicaster/zapp-platform-android/pull/69), [#70](https://github.com/applicaster/zapp-platform-android/pull/70)
+- Tablet support added [71](https://github.com/applicaster/zapp-platform-android/pull/71),[72](https://github.com/applicaster/zapp-platform-android/pull/72), [71](https://github.com/applicaster/zapp-platform-android/pull/71),[73](https://github.com/applicaster/zapp-platform-android/pull/73)
```
Comment (ranmeirman) [inline CHANGELOG.md:17]: Can you explain what here? the tablet support was already there, it is the orientation that was added isn't it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+- CI application updated [#74](https://github.com/applicaster/zapp-platform-android/pull/74)
+- QuickBrick updated to 4.1.4 [#81](https://github.com/applicaster/zapp-platform-android/pull/81)
```
Comment (ranmeirman) [inline CHANGELOG.md:19]: this should probably be the first line

**Review by ranmeirman — CHANGES_REQUESTED** [review]
Please see comments

File: CHANGELOG.md (hunk 3 above)
Comment (inckie) [reply CHANGELOG.md:15]: Updated. I put entries in merge order (usually same as PR#, except for old ones), but I made sorting descending now

File: CHANGELOG.md (hunk 2 above)
Comment (inckie) [reply CHANGELOG.md]: Mostly yes, updated

**Review by ranmeirman — CHANGES_REQUESTED** [review]
so we bumped the version to 2, can you add a breaking changes section?

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**